### ISSUE TITLE
In-Game: [YYC] Silent crash starting in runtime 2024.1100.0.665

### ISSUE THREAD

**Issue by shichen85** [YoYoGames/GameMaker-Bugs#7976]
### Description
The change between 2024.1100.0.664 and 2024.1100.0.665 is causing silent crashes for YYC.

```
Running test # 2 
[running] chained_custom_constructor_test 
func: Foo

C:\temp\GMS2TEMP\Ganary_25A83BDC_YYC\Ganary.exe exited with non-zero status (-1073741819)
elapsed time 00:00:25.5298425s for command "C:\ProgramData\GameMakerStudio2-Beta\Cache\runtimes\runtime-2024.1100.0.665/bin/igor/windows/x64/Igor.exe" -j=8  -options="C:\temp\GMS2TEMP\build.bff" -v -- Windows Run started at 10/11/2024 09:55:24
FAILED: Run Program Complete
```

### Steps To Reproduce
1. Run the sample for Windows YYC underd the `dev` config
2. See crash

### Which version of GameMaker are you reporting this issue for?
IDE v2024.800.0.618 Runtime v2024.1100.0.665

### Which operating system(s) are you seeing the problem on?
Windows 10.0.22000.0

### Which platform(s) are you seeing the problem on?
Windows

[5d6a997d-cf3e-4536-a404-44aa159de6d9](https://api.gamemaker.io/api/github/downloads/5d6a997d-cf3e-4536-a404-44aa159de6d9)

**Comment by Emc1923** [comment]
Issue also present in  IDE v2024.8.1.171  Runtime v2024.8.1.218, does not happen with v2024.1100.0.664 but is happens in .665/.666

**Comment by shichen85** [comment]
Still happening in runtime 670.

**Comment by rwkay** [comment]
This appears to be fixed after fix for https://github.com/YoYoGames/GameMaker/issues/4447 has been merged in...

**Comment by sihammill** [comment]
Fix verified in IDE v2024.1100.0.653  Runtime v2024.1100.0.679
